### PR TITLE
Test Branch for Git Branches Session

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -12,3 +12,6 @@
 - `git clone <ssh link>`
 
 ## Git Branches & PRs
+
```
Comment (ahohnsen) [inline Readme.md:15]: Is this empty line really necessary? I recommend to delete it.

File: Readme.md [hunk 2]
```diff
@@ -12,3 +12,6 @@
...
+- `git switch -c <branch-name>`
+- `git switch <branch-name>`
```
Comment (ahohnsen) [inline Readme.md:17]: I believe you do not need the "-c".

```suggestion
- `git switch <branch-name>`
- `git switch <branch-name>`
```

**Review by ahohnsen — COMMENTED** [review]
Well done, I have a few comments.

File: Readme.md (hunk 1 above)
Comment (ImMrSage) [reply Readme.md:15]: I think it is indeed necessary

Comment (uschihofmann) [reply Readme.md:15]: Thanks for the compliment

Comment (denoerss) [reply Readme.md:15]: @ahohnsen  Yes, this is for clarity :)

File: Readme.md (hunk 2 above)
Comment (TastyCrysis) [reply Readme.md:17]: I think the "-c" is important.

File: Readme.md (hunk 1 above)
Comment (jnuck) [reply Readme.md:15]: Looks good to me 👍🏼

File: Readme.md (hunk 2 above)
Comment (ImMrSage) [reply Readme.md:17]: -c stands for create

File: Readme.md (hunk 1 above)
Comment (Silver-SEO) [reply Readme.md:15]: I think we need to delete the complete project.

File: Readme.md (hunk 2 above)
Comment (G-Kerber) [reply Readme.md:17]: @TastyCrysis is a good idea

**Comment by ImMrSage** [comment]
I think the changes aren`t necessary

**Pull request merged**